types/types.go

```diff
@@ -256,7 +256,10 @@ type Info struct {
 	Runtimes           map[string]Runtime
 	DefaultRuntime     string
 	Swarm              swarm.Info
-	LiveRestore        bool
+	// Determine whether containers should be kept running when the
```

Comment:
I meant: the go official examples themselves don't enforce this for struct fields. The example I put in my previous comment, is straight from the go guideline you've linked. 

And yes, this is not enforced for struct fields in our repos or by golint for that matter (just look 10 lines down).

Anyhow, I'll change it.